charger/evsedin.go

```diff
@@ -2,31 +2,34 @@ package charger
 
 import (
 	"context"
+	"encoding/binary"
 	"fmt"
 	"time"
 
 	"github.com/evcc-io/evcc/api"
 	"github.com/evcc-io/evcc/util"
 	"github.com/evcc-io/evcc/util/modbus"
+	"github.com/volkszaehler/mbmd/encoding"
 )
 
 // EvseDIN charger implementation
 type EvseDIN struct {
 	conn    *modbus.Connection
-	current int64
+	current uint16
 }
 
 const (
-	evseRegAmpsConfig    = 1000
-	evseRegVehicleStatus = 1002
+	evseRegCurrent  = 1000
+	evseRegStatus   = 1002
+	evseRegFirmware = 1005
+	evseRegConfig   = 2005
 )
 
 func init() {
-	registry.AddCtx("simpleevse", NewEvseDINFromConfig) // deprecated
```

Comment:
Breaking change

Reply:
Isso, aber schadet ja nix nett zu sein. Oder einfach nicht ohne Not machen ;) Beworben wirds ja nicht…